evaluation/loadenv.sh

```diff
@@ -0,0 +1,18 @@
+ #!/bin/sh
+
+echo ""
+echo "Loading azd .env file from current environment"
+echo ""
+
+while IFS='=' read -r key value; do
+    value=$(echo "$value" | sed 's/^"//' | sed 's/"$//')
+    export "$key=$value"
+done <<EOF
+$(azd env get-values)
```

Comment:
Maybe use the azd env get-values > .env approach instead. Call that out in the README.

Also make a .env.sample

So more compatible with non-azd users.